src/v2/view-builder/internals/BaseForm.js

```diff
@@ -100,6 +100,23 @@ export default Form.extend({
     }
   },
 
+  /*
+   * Whether an error message should be rendered as the default BaeForm error banner or not
+   * if this fn returns true, BaseForm won't render the error banner, and the Form needs to
+   * override showCustomErrorMessage with the logic to render an error message
+   */
+  isErrorMessageCustomized() {
+    return false;
+  },
+
+  /*
+   * Renders an error message
+   * This should be overriden if isErrorMessageCustomized() returns true
+   */
+  showCustomErrorMessage() {
```

Comment:
not should we name this  showCustomFormErrorMessage just to clearly differentiate form showMessages. 

Also add a comment before showMessages " this function is called only on initialize and displays messages only on view load and not after that.